index.html

```diff
@@ -11,13 +11,14 @@
 </head>
 <body>
     <header>
-        <a href="index.html" class="logo"><img src="assets/images/Logo Placeholder.png" alt="Portfolio Logo"></a>
+        <a id="logo" href="index.html"><img src="assets/images/Logo Placeholder.png" alt="Portfolio Logo"></a>
         <nav>
-            <a href="#portfolio" class="nav-desktop">Portfolio</a>
-            <a href="#about" class="nav-desktop">About</a>
-            <a href="#contact" class="nav-desktop">contact</a>
-            <img class="nav-desktop" src="assets/images/Icon - Mail.png" alt="Contact icon">
-            <img class="nav-mobile" src="assets/images/Union.png" alt="Menu icon">
+            <a class="nav nav-desktop hide" href="#portfolio">Portfolio</a>
+            <a class="nav nav-desktop hide" href="#about">About</a>
+            <a class="nav nav-desktop hide" href="#contact">contact</a>
+            <img class="nav nav-desktop hide" src="assets/images/Icon - Mail.png" alt="Contact icon">
+            <img id="menu-icon" class="nav nav-mobile" src="assets/images/Menu.png" alt="Menu icon">
+            <img id="exit-icon" class="nav nav-mobile hide" src="assets/images/Exit.png" alt="Exit icon">
         </nav>
```

Comment:
- [ ] Dear @jlvFlores, Your design is looking great already. But I believe there are some improvements we can make. as you can see we need the `underline` under each menu item, the `white space` on the bottom, and the message icon should not be visible on the mobile menu.


### Figma design 
![Screenshot 2023-01-31 194816](https://user-images.githubusercontent.com/69077061/215828712-c4dee88e-cc1d-4865-9978-9ba2b44a4f7c.png)

### Your design
![Screenshot 2023-01-31 194915](https://user-images.githubusercontent.com/69077061/215828724-524b8886-5ec9-486e-9325-88f924a72e4f.png)

Reply:
Hello @Tadesse-Alemayehu thank you for your feedback, I will add the changes that you've proposed to me. However, I went over the Figma template and noticed that the white space is part of the background and not in fact an element. I could make the dropdown a bit longer to hide the headline and make it a bit better to look at, what do you think?

Reply:
Noce to reach out @jlvFlores, But the white part is not the background, if you notice there is a horizontal line in the middle that is going to be used to close the menu (just like the `X` close the menu). You are not expected to implement that functionality as of now, but the design should be there.